G1ANT.Addon.OCR.Tesseract/G1ANT.Addon.OCR.Tesseract/Commands/OcrOfflineFindCommand.cs

```diff
@@ -37,31 +38,46 @@ public class Arguments : CommandArguments
 
             [Argument(Tooltip = "The language which should be considered trying to recognize text")]
             public TextStructure Language { get; set; } = new TextStructure("eng");
+
+            [Argument(Tooltip = "The ratio used for rescaling image before OCR. Values above 1.0")]
```

Comment:
You don't verify the value you mentioned (above 1.0). Actually, what's wrong with using values < 1?

"The ratio used for rescaling of the image before doing actual OCR. Default is 2.0. Higher values are better for recognizing details like small fonts."

Reply:
values below 1.0 are just worse than original image